django/contrib/admin/helpers.py

```diff
@@ -304,6 +308,14 @@ def inline_formset_data(self):
         })
 
     @property
+    def forms(self):
+        return self.formset.forms
+
+    @property
+    def non_form_errors(self):
+        return self.formset.non_form_errors
```

Comment:
No, I wouldn't expect that. Maybe the diff doesn't make it clear, but this attribute is on `InlineAdminFormSet` not `AdminForm`.